pangeo_forge_esgf/recipe_inputs.py

```diff
@@ -140,6 +146,9 @@ async def _esgf_api_request(
     if params["latest"] == "true":
         if "version" in facets:
             del facets["version"]
```

Comment:
I have previously wondered that this seems yanky. If I set "latest" to false in the requests I can actually provide the version. Is that behavior the same for CORDEX?

Reply:
I don't really know.. I think in CORDEX, datasets are always the "latest" version while retracted versions are removed from ESGF...